README.md

```diff
@@ -1,36 +1,52 @@
-# Friday Night Funkin
+# Friday Night Funkin!
 
-This is the repository for Friday Night Funkin, a game originally made for Ludum Dare 47 "Stuck In a Loop".
+![FNF_Logo](https://user-images.githubusercontent.com/72235309/112068307-08fe8d80-8b40-11eb-84d0-fc64a10dd845.png "FNF Logo")
 
-Play the Ludum Dare prototype here: https://ninja-muffin24.itch.io/friday-night-funkin
-Play the Newgrounds one here: https://www.newgrounds.com/portal/view/770371
-Support the project on the itch.io page: https://ninja-muffin24.itch.io/funkin
+### This is the repository for Friday Night Funkin, a game originally made for Ludum Dare 47 "Stuck In a Loop".
 
-## Credits / shoutouts
+Play the Ludum Dare prototype [here](https://ninja-muffin24.itch.io/friday-night-funkin)!
 
-- [ninjamuffin99 (me!)](https://twitter.com/ninja_muffin99) - Programmer
-- [PhantomArcade3K](https://twitter.com/phantomarcade3k) and [Evilsk8r](https://twitter.com/evilsk8r) - Art
-- [Kawaisprite](https://twitter.com/kawaisprite) - Musician
+Play the Newgrounds one [here](https://www.newgrounds.com/portal/view/770371)!

```

Comment:
I would place the Newgrounds one at the top as many people would want to play this one rather than the Ludum Dare one.